src/main/java/net/dv8tion/jda/api/utils/MiscUtil.java

```diff
@@ -209,4 +213,8 @@ public static void appendTo(Formatter formatter, int width, int precision, boole
             throw new UncheckedIOException(e);
         }
     }
+
+    public static <T> Collector<T, ?, List<T>> toUnmodifiableList() {
```

Comment:
This should probably be inside of `Helpers` instead, where we already have other collection utils.
```suggestion
    public static <T> Collector<T, ?, List<T>> toUnmodifiableList()
    {
```